### PR TITLE
Bug fix create room form

### LINKED ISSUE #32: Fix a bug in "create room" form

### PR DESCRIPTION
**Pull request by Mariphy**
Hi, I noticed that after "create room" form submission it redirects to the 404 page, so I fixed the routes

### CODE REVIEW

**Review by dan2811 — DISMISSED** [review]
Looks good to me! Be sure to merge in order, so wait for the `add delte room procedure` PR to be merged before merging this one.

**Comment by Mariphy** [comment]
Hi Dan, I see your previous approval, but for some reason I still get "review required", could you review again, please?

**Review by dan2811 — APPROVED** [review]

**Pull request merged**
